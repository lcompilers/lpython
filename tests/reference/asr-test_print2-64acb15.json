{
    "basename": "asr-test_print2-64acb15",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_print2.py",
    "infile_hash": "af07d17d5b3f16db024a3e893923988349c8f6c8bf1a6c5507d56a6d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_print2-64acb15.stderr",
    "stderr_hash": "e92bba85b957e7034c5172981b3b27ed7b3f0ac62167d82175890bc9",
    "returncode": 2
}