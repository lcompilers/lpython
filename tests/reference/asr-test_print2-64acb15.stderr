semantic error: End is expected to be of string type
 --> tests/errors/test_print2.py:2:26
  |
2 |      print("a", "b", end=1)
  |                          ^ Expected string, found: integer
